### DISCUSSION TITLE
i want to create my own website

### DISCUSSION THREAD

**Discussion by getsisay** [community/community#129042] [Programming Help]
i want to create my own website

**Comment by getsisay** [comment]
[Comminutors.pptx](https://github.com/user-attachments/files/15897301/Comminutors.pptx)

**Comment by ebndev** [comment]
Hi @getsisay, thanks for being a part of the GitHub Community! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction.

**Comment by mensahTribeWeb** [comment]
@getsisay go for it, building will always be rewarding #happycoding 🫡

**Comment by manuel14mds** [comment]
Type that in ChatGPT and explain it what you want.
